src/banks/registries/directory.py

```diff
@@ -43,19 +50,42 @@ def _scan(self):
         self._index_path.write_text(self._index.model_dump_json())
 
     def get(self, *, name: str, version: str | None = None) -> "Prompt":
-        version = version or "0"
+        version = version or DEFAULT_VERSION
         for pf in self._index.files:
             if pf.name == name and pf.version == version and pf.path.exists():
                 return Prompt(pf.path.read_text())
         raise TemplateNotFoundError
 
     def set(self, *, name: str, prompt: Prompt, version: str | None = None, overwrite: bool = False):
-        version = version or "0"
+        version = version or DEFAULT_VERSION
         for pf in self._index.files:
             if pf.name == name and pf.version == version and overwrite:
                 pf.path.write_text(prompt.raw)
                 return
-        new_prompt_file = self._path / "{name}.{version}.jinja"
+        new_prompt_file = self._path / f"{name}.{version}.jinja"
         new_prompt_file.write_text(prompt.raw)
         pf = PromptFile(name=name, version=version, path=new_prompt_file)
         self._index.files.append(pf)
+
+    def get_meta(self, *, name: str, version: str | None = None) -> dict:
+        version = version or DEFAULT_VERSION
+        meta_path = self._meta_path / f"{name}.{version}.json"
+        if not meta_path.exists():
+            msg = f"Meta directory or file for prompt {name}:{version}.jinja not found."
+            raise FileNotFoundError(msg)
+        return json.loads(open(meta_path, encoding="utf-8").read())
+
+    def set_meta(self, *, meta: dict, name: str, version: str | None = None, overwrite: bool = False):
```

Comment:
I would try to pass meta to the regular `set()` (see comment above)

Reply:
I would do that but I will also provide `set_meta` separately because there are many use cases (i.e., adding accuracy after running the prompt, adding failed stats) and we also need such things per prompt so I will still keep the different folders for storing the meta and the prompt/index.